CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# v1.3.6
+
+### `@liveblocks/react`
+
+- Implement request deduping and handle race conditions during mutations and
+  revalidation, inspired by SWR.
+
```

Comment:
It should be part of `v.1.3.5` since it hasn't been released yet. Also, I think we should mention Comments since it lives within `@liveblocks/react`, and maybe we can keep the SWR credit in the code.

```
- Deduplicate Comments requests and improve how race conditions are handled during mutations.
```

Reply:
Yeah, this is in part my bad. I used to add `# v1.3.5 (not released yet)` in the heading which made that easy to recognize to avoid just this, but I've stopped doing that because these days it's a bit too annoying to open a PR right before a release just to remove that `(not released yet)` part of the heading, getting CI to pass, getting approval, etc. Maybe we could set things up in a way that would allow us to have a free-to-merge pass on PRs that only change README or CHANGELOG files and still remain SOC2-compliant?

Reply:
I also stopped doing this (I used to add it as `# Unpublished`) for the same reason, it would be nice to have a way to quickly merge these types of PRs yeah. 🙏